source/includes/harvest/_applications.md

```diff
@@ -424,3 +424,95 @@ id | ID of the application to unreject
 ### JSON Body Parameters
 
 No JSON body parameters
+
+
+## Update an application
+
+```shell
+curl -X PATCH 'https://harvest.greenhouse.io/v1/applications/{id}/"
+-H "On-Behalf-Of: {greenhouse user ID}"
+-H "Authorization: Basic MGQwMzFkODIyN2VhZmE2MWRjMzc1YTZjMmUwNjdlMjQ6"
+```
+
+> The above command takes a JSON request, structured like this:
+
+```json
+{
+  "rejection_reason_id": 815,
+  "notes": "The candidate is not qualified for this position.",
+  "source_id": 16
+}
+```
+
+> The above returns a JSON response, structured like this:
+
+```json
+{
+  "id": 985314,
+  "candidate_id": 978031,
+  "prospect": false,
+  "applied_at": "2014-03-26T20:11:39.000Z",
+  "last_activity_at": "2014-03-27T16:13:15.000Z",
+  "source": {
+    "id": 1871,
+    "public_name": "Happy Hour"
+  },
+  "credited_to": {
+    "id": 4080,
+    "name": "Kate Austen"
+  },
+  "jobs": [
+    {
+      "id": 123,
+      "name": "Accounting Manager"
+    }
+  ],
+  "status": "rejected",
+  "current_stage": {
+    "id": 62828,
+    "name": "Recruiter Phone Screen"
+  }
+}
+```
+
+Update this application. The response is populated with the application's information which will reflect its new state.
+
+### HTTP Request
+
+`PATCH https://harvest.greenhouse.io/v1/applications/{id}`
+
+### Headers
+
+Header | Description
+--------- | -----------
+On-Behalf-Of | ID of the user issuing this request. Required for auditing purposes.
+
+
+### JSON Body Parameters
+
+Parameter | Required | Type | Description
+--------- | ----------- | ----------- | ----------- | -----------
+related_post_id | No | integer | The ID of the application's related post
+related_post_type | No | integer | The type of the application's related post
+source_id | No | integer | The ID of the application's source
+stage_status_id | No | integer | The ID of the application's stage status
+candidate_rejection_reason_id | No | integer | The ID of the application's candidate's rejection reason
+current_application_stage_id | No | integer | The ID of the application's current stage
+rejection_note_id | No | integer | The ID of the application's rejection note
+rejected_at | No | datetime | The date and time at which the application was rejected
+needs_decision | No | boolean | Boolean indicating whether the application needs a hiring decision
+referrer_id | No | integer | The ID of candidate's referer, if applicable
+prospect | No | boolean | A boolean indicating whether this application represents a candidate or a prospect
```

Comment:
Or, "A boolean indicating whether this application represents a prospect instead of a candidate"